### PR TITLE
Add instructions to integration_test example README

### PR DESCRIPTION
**Pull request by johnpryan**
Adds Chromedriver instructions to the integration_test example's README

This was previously submitted to flutter/plugins but now integration_test is in flutter/flutter: https://github.com/flutter/plugins/pull/3289

### CODE REVIEW

File: packages/integration_test/example/README.md [hunk 1]
```diff
@@ -14,6 +14,15 @@ flutter drive \
 
 Web:
 
+In one shell, run Chromedriver ([download
+here](https://chromedriver.chromium.org/downloads)):
+
+```
+chromedriver --port 8444
```
Comment (stuartmorgan) [inline packages/integration_test/example/README.md:21]: Isn't the default 4444? That's what I've previously seen things use, so assumed was a hard-code default.

Comment (johnpryan) [reply packages/integration_test/example/README.md:21]: When I run it, it's `9515` 🤷

File: packages/integration_test/example/README.md [hunk 2]
```diff
@@ -14,6 +14,15 @@ flutter drive \
 
 Web:
```
Comment (ditman) [inline packages/integration_test/example/README.md:15]: I'd make the Android/iOS and Web proper headers:

```suggestion
## Web
```

File: packages/integration_test/example/README.md (hunk 1 above)
Comment (ditman) [reply packages/integration_test/example/README.md:21]: Hahaha --port 4444 is what I've been using for a while now; it probably doesn't matter much as long as `chromedriver` starts in an unoccupied port.

@johnpryan this is what I've come up with for our web integration tests:

https://github.com/flutter/plugins/blob/master/packages/connectivity/connectivity_for_web/example/README.md

**Review by ditman — APPROVED** [review]
LGTM, this docs are welcome. 

I'd suggest this documentation also linked to this page:

https://flutter.dev/docs/testing/integration-tests#running-in-a-browser

This is where the port 4444 probably comes from, and it has interesting notes about project structure as well.

**Comment by goderbauer** [comment]
Is this ready to be submitted?

**Comment by johnpryan** [comment]
Yes, I think so.

**Pull request merged**